java/src/main/java/com/google/appengine/tools/pipeline/JobSetting.java

```diff
@@ -191,4 +191,18 @@ public StatusConsoleUrl(String statusConsoleUrl) {
       super(statusConsoleUrl);
     }
   }
+
+  /** 
+   * A setting specifying the job's display name. This will replace 
```

Comment:
I don't think the second statement is needed (if you insist, make it more clear and period).